### DISCUSSION TITLE
Some Animations are disappeared after migration ng-zorro from v9 to v10

### DISCUSSION THREAD

**Discussion by enk0de** [NG-ZORRO/ng-zorro-antd#6014] [Q&A]
hi. i am a front-end developer in south-korea.
i recently upgraded ng-zorro version from v9 to v10.
and after that, i have got an ma----ny problems.

one of them, most seriously, is that **SOME ANIMATION IS DISAPPEARED**

when we changes another tab in tabset, we can see the changing animation.
but in my app, it isn't work!

futhermore, this is shown in modal animation. (the showing modal animation is disappeared or became slowest)

how can i solve this problems?
i will be so thankful if u let me know the solutions.

thank you community guys.

**Comment by wenqi73** [comment]
Could you please a reproduction by stackblitz? The tabs animation works fine on the official site.
